static/src/js/components/TemplateList/__tests__/TemplateList.test.jsx

```diff
@@ -9,7 +9,7 @@ import errorLogger from '@/js/utils/errorLogger'
 import getTemplates from '@/js/utils/getTemplates'
 import deleteTemplate from '@/js/utils/deleteTemplate'
 import NotificationsContext from '@/js/context/NotificationsContext'
-import MMT_COOKIE from '@/js/constants/mmtCookie'
+import MMT_COOKIE from '../../../../../../sharedContstants/mmtCookie'
```

Comment:
This should use the @/js/constants way of doing things (absolute path) rather than relative paths (../../../ etc)